references/regex.json

```diff
@@ -1,3 +1,3 @@
 {
-  "submitter_id": "\\b(?!([Dd][Oo])|([Ss][Pp])|([Ss][Aa])|([Tt][Rr])|([pP][Dd])|([Ff][Uu]))\\b^[A-Za-z0-9\\-\\._]{1,64}"
```

Comment:
I remember we were happy to get this to work

Reply:
i know :(